diff --git a/src/Makefile.am b/src/Makefile.am
index 4dbe93fee..e61376795 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -32,12 +32,8 @@ if RELEASE
 %C%_libopenocd_la_CPPFLAGS += -DRELSTR=\"\"
 %C%_libopenocd_la_CPPFLAGS += -DGITVERSION=\"\"
 else
-%C%_libopenocd_la_CPPFLAGS += -DRELSTR=\"`$(top_srcdir)/guess-rev.sh $(top_srcdir)`\"
-%C%_libopenocd_la_CPPFLAGS += -DGITVERSION=\"`cd $(top_srcdir) && git describe`\"
-%C%_libopenocd_la_CPPFLAGS += -DPKGBLDDATE=\"`DATE_FMT=+%F-%R; \
-	SOURCE_DATE_EPOCH="$${SOURCE_DATE_EPOCH:-$$(date +%s)}"; \
-	date -u -d "@$$SOURCE_DATE_EPOCH" "$$DATE_FMT" 2>/dev/null || \
-	date -u -r "$$SOURCE_DATE_EPOCH" "$$DATE_FMT" 2>/dev/null || date -u "$$DATE_FMT"`\"
+%C%_libopenocd_la_CPPFLAGS += -DRELSTR=\"@RELSTR@\"
+%C%_libopenocd_la_CPPFLAGS += -DGITVERSION=\"@GITVERSION@\"
 endif

 # add default CPPFLAGS
